lsst/ts/laser/csc.py

```diff
@@ -102,6 +132,9 @@ def assert_propagating(self, action):
         """
         self.assert_enabled("changeWavelength")
         self.model.change_wavelength(id_data.data.wavelength)
+        wavelength_changed_topic = self.evt_wavelengthChanged.DataType()
+        wavelength_changed_topic.wavelength= id_data.data.wavelength
+        self.evt_wavelengthChanged.put(wavelength_changed_topic)
 
     async def do_startPropagateLaser(self,id_data):
```

Comment:
Another method defined as a coroutine but with no `await` statements on it. See previous comment.